src/libostree/ostree-repo-checkout.c

```diff
@@ -516,24 +458,21 @@ checkout_one_file_at (OstreeRepo                        *repo,
                                            destination_dfd, destination_name,
                                            TRUE, &hardlink_res,
                                            cancellable, error))
-                goto out;
+                return FALSE;
 
               if (hardlink_res == HARDLINK_RESULT_LINKED && options->devino_to_csum_cache)
                 {
                   struct stat stbuf;
                   OstreeDevIno *key;
-                  
+
                   if (TEMP_FAILURE_RETRY (fstatat (destination_dfd, destination_name, &stbuf, AT_SYMLINK_NOFOLLOW)) != 0)
```

Comment:
Re. your comment, I think my vote is for `TEMP_FAILURE_RETRY`. I find it easier to read (plus, it's even less boilerplate!).